versions.gradle

```diff
@@ -69,7 +68,7 @@ def android_components = [:]
 android_components.telemetry = "org.mozilla.components:service-telemetry:$versions.android_components"
 android_components.glean = "org.mozilla.components:service-glean:$versions.android_components"
 android_components.browser_errorpages = "org.mozilla.components:browser-errorpages:$versions.android_components"
-android_components.browser_search = "org.mozilla.components:browser-search:$versions.android_components"
+android_components.browser_search = "org.mozilla.components:browser-search:75.0.23"
```

Comment:
Yeah, browser search suffered a huge revamp upstream. This was actually one of the things that have been blocking us from upgrading AC for a while. Perhaps this approach is better, let's upgrade the others and tackle the search component later.

Reply:
According to the RFC here: https://github.com/mozilla-mobile/firefox-android/blob/main/docs/rfcs/0002-browser-search-rewrite.md browser-search was replaced by feature-search Will update accordingly so that we can keep everything up to date.

Reply:
Okay, there are quite a few things that need refactoring if we want this change. So I opened a new issue for this and let's keep browser-search for now. Issue opened: https://github.com/Igalia/wolvic/issues/841